src/Definition/ConfigProcessor/AclConfigProcessor.php

```diff
@@ -73,13 +73,13 @@ public function process(LazyConfig $lazyConfig): LazyConfig
     private static function findFieldResolver(array $field, ResolveInfo $info, callable $defaultResolver): callable
     {
         if (isset($field['resolve'])) {
-            $resolver = $field['resolve'];
-        } elseif (isset($info->parentType->config['resolveField'])) {
-            $resolver = $info->parentType->config['resolveField'];
-        } else {
-            $resolver = $defaultResolver;
+            return $field['resolve'];
+        } 
```

Comment:
```suggestion
        }
```

Fix CS: remove trailing whitespace